vendor.conf

```diff
@@ -36,28 +36,30 @@ github.com/Microsoft/go-winio                       v0.4.14
 github.com/Microsoft/hcsshim                        v0.8.9
 github.com/opencontainers/go-digest                 v1.0.0
 github.com/opencontainers/image-spec                v1.0.1
-github.com/opencontainers/runc                      v1.0.0-rc10
-github.com/opencontainers/runtime-spec              v1.0.2
+github.com/opencontainers/runc                      v1.0.0-rc91
+github.com/opencontainers/runtime-spec              237cc4f519e2e8f9b235bacccfa8ef5a84df2875
```

Comment:
kk pickup for errno fix.

Reply:
errno of what?

Reply:
https://github.com/opencontainers/runtime-spec/commit/023044323086583e5b0071ed1e6c18fd499836f3#diff-7f24d60f0cbb9c433e165467e3d34838R672